hibernate-core/src/test/java/org/hibernate/test/fileimport/MultiLineImportExtractorTest.java

```diff
@@ -46,15 +46,15 @@ public void testExtraction() throws IOException {
 
 				assertThat( commands[1], is( "INSERT INTO test_data VALUES (1, 'sample')" ) );
 
-				assertThat( commands[2], is( "DELETE" + System.lineSeparator() + "  FROM test_data" ) );
+				assertThat( commands[2], is( "DELETE   FROM test_data" ) );
```

Comment:
so now we convert a newline into a standard whitespace? Was the intention not to also emit multi-line SQL, possibly for better logging?

Reply:
the issue is when we have something like
 ```
DELETE
FROM TEST DATA
```
where there is no space after `DELETE` and no space before `FROM`. by the way the `System.lineSeparator()` was introduced by the previous PR.

Reply:
probably another solution is to keep the grammar as it was before the  before https://github.com/hibernate/hibernate-orm/pull/3502 change and in some way detect and add the space on case it it needed in the `SqlScriptParser.out()` method.

Reply:
> so now we convert a newline into a standard whitespace? Was the intention not to also emit multi-line SQL, possibly for better logging?

I wonder what happens if hibernate.format_sql is enables when you use a multiline script? Do you get rubbish logging with random-looking newlines?

Reply:
(Before this latest change, I mean.)

Reply:
I don't know, I should try

Reply:
I might be wrong, but I don't think we apply SQL formatting on the import scripts?